source/documentation/breaking-changes/calc-functions.md

```diff
@@ -0,0 +1,121 @@
+
+These calc functions are now reserved words, which means any custom defined
+functions defined with: [`round()`], [`mod()`], [`rem()`], [`sin()`], [`cos()`], [`tan()`],
+[`asin()`], [`acos()`], [`atan()`], [`atan2()`], [`pow()`], [`sqrt()`], [`hypot()`],
+[`log()`], [`exp()`], [`abs()`], or [`sign()`] will be parsed as calculations
+instead of functions.
+
+[`round()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/round
+[`abs()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/abs
+[`sin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sin
+[`cos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/cos
+[`tan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/tan
+[`asin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/asin
+[`acos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/acos
+[`atan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan
+[`atan2()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan2
+[`pow()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/pow
+[`sqrt()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sqrt
+[`hypot()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/hypot
+[`log()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/log
+[`exp()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/exp
+[`mod()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/mod
+[`rem()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/rem
+[`sign()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sign
+
+{% codeExample 'calc-function-bad', false %}
+@function rem($number) {
+  @return $number % 10;
+}
+
+/* This example would parse `rem()` as a calc function, not as the declared
+function above. Since `rem()` requires two parameters, this would throw a
+parameter error. */
+.sidebar {
+  margin-left: rem(9) * 1px;
+}
+
+===
+@function rem($number)
+  @return $number % 10
+
+// This example would parse `rem()` as a calc function, not as the declared
+// function above. Since `rem()` requires two parameters, this would throw a
+// parameter error.
+.sidebar 
+  margin-left: rem(9) * 1px
+
+{% endcodeExample %}
+
+However, if the custom `rem()` function was imported to another file, it would
+be parsed as the declared function since it's assigned to a different namespace.
+
+{% codeExample 'calc-function-import', false %}
+<===> foo.scss
+@function rem($number) {
+  @return $number % 10;
+}
+
+<===> main.scss
+// This example would not throw an error.
+@use 'foo';
+
+.sidebar {
+  a: foo.rem(9) * 1px;
+}
+===
+<===> foo.scss
+@function rem($number)
+  @return $number % 10
+
+<===> main.scss
+// This example would not throw an error.
+@use 'foo'
+
+.sidebar
+  a: foo.rem(9) * 1px
+
+{% endcodeExample %}
+
+{% headsUp %}
+  If the module was imported [_without a namespace_] 
+  calc functions would override the custom declared functions.
+  For instance, `@use 'foo' as *` would change the call to `a: rem(10)`,
+  which would result in an error.
```

Comment:
```suggestion
  Because calling a function from a module loaded [_without a namespace_]
  by writing `@use <url> as *` has the same syntax as calling a global function,
  Sass will parse these function calls as calculations, not as calls to the
  function defined in the module!
```